apps/ui/src/components/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import { ConfigProvider, Flex, Image, Layout, Typography } from 'antd';
+
+import { pcglFooterTheme } from '@/components/providers/ThemeProvider';
+
+import PCGLFOOTER from '@/assets/pcgl-logo-footer.png';
+
+const { Footer } = Layout;
+const { Text, Link } = Typography;
+
+const linkStyle: React.CSSProperties = {
+	textAlign: 'center',
+	textWrap: 'nowrap',
+};
+
+const textStyle: React.CSSProperties = {
+	textAlign: 'center',
+};
+
+const FooterComponent = () => {
+	return (
+		<ConfigProvider theme={pcglFooterTheme}>
+			<Footer>
+				<Flex justify="center" align="center">
+					<Link target="_blank">
+						<Image width={200} src={PCGLFOOTER} preview={false} />
+					</Link>
+					<Flex flex={1} vertical justify="center" align="center" gap={10} wrap>
+						<Flex gap={20} justify="center" align="center" wrap>
+							<Link style={linkStyle} underline target="_blank">
```

Comment:
these links that open in a new tab should also have the `rel` attribute set, as described in this article: https://rainey.dev/target-_blank-noopener-noreferrer-and-nofollow/

...which leads me to suggest they can all be abstracted into a reusable `LinkToBlankTarget` component (you choose a better name 😆), to DRY things up a little.

Reply:
Yeah was going to suggest we could `.map` these

Reply:
@justincorrigible I believe ant design has got that covered for us under the hood with their Link component!

_Example from our localhost html:_ 
```
<a class="ant-typography css-dev-only-do-not-override-mnm4b2" target="_blank" rel="noopener noreferrer" style="min-width: 100px; text-align: center;">Policies &amp; Guidelines</a>

```

Reply:
> Yeah was going to suggest we could `.map` these

I can map these elements